### DISCUSSION TITLE
[Bug] Get JSON Data from endpoint API page in Grafana - https://status.snowflake.com/

### DISCUSSION THREAD

**Discussion by eugenesheyko** [grafana/grafana-infinity-datasource#597] [Q&A] [answered]
Hello,

We have Grafana running on AWS v9.1.7 and have installed the Infinity plugin though getting the following error -
![image](https://github.com/yesoreyeram/grafana-infinity-datasource/assets/19654539/835e03c7-914f-4c3c-8316-e65e0f9aaf4f)
`Plugin disabled due to missing plugin signature`

And thus not able to add the data source for it.

Please let us know how we could re-install the plugin and add the particular data source for it. 

The problem we're trying to achieve is that we're trying to display data from the following endpoint - https://status.snowflake.com/.

For example, wondering if we could scrape the API endpoint and be able to scrape this data. -
https://status.snowflake.com/api/v2/summary.json

**Comment by yesoreyeram** [comment]
How you installed the plugin? Seems the plugin wasn't installed correctly.

**Reply by eugenesheyko** [reply, marked as answer]
Have resolved this by updating the plugin custom settings on our end. 
`allow_loading_unsigned_plugins = yesoreyeram-infinity-datasource`

Please refer to the Grafana community for more details -
https://community.grafana.com/t/usigned-plugin-is-not-showing-in-gui-plugins/55256

Thank you
